can_tools/scrapers/official/AZ/AZ_TableauPull.py

```diff
@@ -157,3 +131,9 @@ def normalize(self, data):
             "value",
         ]
         return out.loc[:, cols_to_keep]
+
+
+if __name__ == "__main__":
+    d = ArizonaData()
+    data = d.fetch()
+    df = d.normalize(data)
```

Comment:
not sure if you want to make this a command line tool - i'm finding myself do things like:
```
python -m can_tools.scraper_maintenance latest-vaccine-data LAVaccineCounty  -d 1
```
all the time now :)

realize that this is a slight variation on that

Reply:
but I think whatever that looks like could/probably should be in another pr

Reply:
Ahh good catch, I forgot I didn't delete this one :laughing: